universal-login-contracts/test/contracts/ERC1077.js

```diff
@@ -319,6 +333,13 @@ describe('ERC1077', async  () => {
         expect(await identity.lastNonce()).to.eq(1);
       });
 
+      it('with 3 required Signature', async () => {
```

Comment:
let's be a bit more precise on naming:
`call method with 3 out of 3 required signatures`

an we can rename two tests above:
`call method with 3 out of 2 required signatures`

Reply:
in my tests the scenario of 3 out of 2 signature doesn't exist.i think it's a good a idea and i will add that test